src/directives/click_action.ts

```diff
@@ -33,7 +33,7 @@ export default [
     ) {
 
         const validateThreemaId = (id: string): boolean => {
-            return id !== undefined && id !== null && /^[0-9A-Z]{8}/.test(id);
+            return id !== undefined && id !== null && /^([a-zA-Z0-9\*]{1}[a-zA-Z0-9]{7})$/.test(id);
```

Comment:
The `{1}` is optional.

```suggestion
            return id !== undefined && id !== null && /^([a-zA-Z0-9\*][a-zA-Z0-9]{7})$/.test(id);
```